plugin/adsk/scripts/mayaUsdMenu.mel

```diff
@@ -274,8 +327,33 @@ global proc mayaUsd_updateSelectionKindMenuItem() {
             global string $gMayaUsdSelectModeSubMenu;
             $menuItems = `menu -q -itemArray $gMayaUsdSelectModeSubMenu`;
             for($i = 0; $i < size($menuItems); $i++) {
-                $label = `menuItem -q -label $menuItems[$i]`;
-                if($label == $selectionKind) {
+                string $label = `menuItem -q -label $menuItems[$i]`;
+                if(`tolower($label)` == $selectionKind) {
+                    menuItem -e -radioButton on $menuItems[$i];
+                    break;
+                }
+            }
+    }
+}
+
+///////////////////////////////////////////////////////////////////////////////
+// mayaUsd_updatePointInstancesPickModeMenuItem
+// Update the menu item radio buttons in the Point Instances Pick Mode menu
+global proc mayaUsd_updatePointInstancesPickModeMenuItem() {
+    // Note: we do all string compares as case-insenstive (lowercase)
+    //       as the menu labels are capitalized.
+    string $pickMode = `tolower(mayaUSD_getPointInstancesPickMode())`;
+    switch($pickMode) {
```

Comment:
I think a if would be clearer, but a switch works.

Reply:
I agree, I only used a switch because I copied the existing proc (SelectionKind) that used a switch.